src/import-data/protected-data-utils.ts

```diff
@@ -25,6 +25,18 @@ export const isProtectedSource = (url: string, filetype?: string): boolean => {
   }
 };
 
+// This method identifies whether an import source is an AnVIL Explorer import
+export const isAnvilImport = (url: string): boolean => {
```

Comment:
#4305 recently updated the type of `importRequest.url`. It's now a [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) instead of a string.